src/palace/manager/feed/annotator/admin.py

```diff
@@ -10,19 +10,37 @@
 
 
 class AdminAnnotator(LibraryAnnotator):
+    REL_SUPPRESS_FOR_LIBRARY = "http://palaceproject.io/terms/rel/suppress-for-library"
```

Comment:
Might be good to add this to notion as well https://www.notion.so/lyrasis/palaceproject-io-terms-namespace-572089bd44404cf395f02b6b78361fe4?pvs=4

Reply:
Done! Thanks for that suggestion.